docs/src/guides/tables.md

```diff
@@ -125,7 +158,38 @@ This guide will show how to create tables, insert data into them, and update the
     tbl = db.create_table("f16_tbl", data, schema=schema)
     ```
 
+=== "Javascript"
+    You can also create LanceDB tables directly from Arrow tables. 
+    LanceDB supports Float16 data type!
+
+    ```javascript
+    import { FixedSizeList,Field,Int32,Schema,Float16, Table as ArrowTable } from 'apache-arrow'
```

Comment:
can we use snippet import of this code example? 
Mostly, i'd like to make sure that we can get this piece of code compiled and tested.